pkgs/by-name/op/open-vm-tools/package.nix

```diff
@@ -39,6 +39,15 @@
   xmlsec,
   withX ? true,
 }:
+let
+  inherit (lib)
+    licenses
+    maintainers
+    makeBinPath
+    optional
+    optionals
+    ;
+in
```

Comment:
IMO we don't want to maintain such lists. Prefixing the functions with lib. is easier especially when rebasing patches.